### PR TITLE
Add normalDist example in PyROOT version

### PR DESCRIPTION
**Pull request by juanfe**

### CODE REVIEW

File: tutorials/math/normalDist.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Tutorial illustrating the new statistical distributions functions (pdf, cdf and quantile)
+##
+## \author Anna Kreshuk
```
Comment (dpiparo) [inline tutorials/math/normalDist.py]: I don't think this will work in our doc. Perhaps Anna can be cited and your name is the only one to have as \author?

**Comment by etejedor** [comment]
Hi @juanfe can you address @dpiparo's and my comments so that we can merge this?

File: tutorials/math/normalDist.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+legend2.AddEntry(ccdfunc, "normal_cdf_c", "l")
+legend2.Draw()
+
+c1.cd(3)
+qfunc.Draw()
+cqfunc.Draw("same")
+legend3 = TLegend(0.315094, 0.633668, 0.695179, 0.881711)
+legend3.AddEntry(qfunc, "normal_quantile", "l")
+legend3.AddEntry(cqfunc, "normal_quantile_c", "l")
+legend3.Draw()
```
[the remark this answers is not in the document]
Comment (juanfe) [reply tutorials/math/normalDist.py]: Yes it works perfect in Python3!

File: tutorials/math/normalDist.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+##
+## \macro_image
+## \macro_code
+##
+## \author Juan Fernando Jaramillo Botero
+
+from ROOT import TF1, TCanvas, TSystem, TAxis, TLegend
+from ROOT import kRed, kGreen, kBlue
+
+
```
Comment (etejedor) [inline tutorials/math/normalDist.py]: Hi @juanfe, thanks for addressing the comments! I have one final comment for what concerns style: when this is translated into a notebook, every group of consecutive code lines will be put into a separate cell. If before some group of code lines (e.g. where I just added this comment) you add one or more comment lines, those lines will be translated into a markdown cell of a notebook.
Therefore, feel free to add comments to explain what is being done in your code, and that will result in a better-looking notebook as well!

**Comment by etejedor** [comment]
@phsft-bot build

**Comment by etejedor** [comment]
@phsft-bot build

File: tutorials/math/normalDist.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dpiparo) [reply tutorials/math/normalDist.py]: great, thanks.

**Comment by dpiparo** [comment]
@phsft-bot build

**Pull request merged**